content/en/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -295,6 +295,39 @@ a Pod is considered ready, see [Container Probes](/docs/concepts/workloads/pods/
 
 Please note that this field only works if you enable the `StatefulSetMinReadySeconds` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
 
+#### PersistentVolumeClaim Policies
+
+{{< feature-state for_k8s_version="v1.23" state="alpha" >}}
+
+`.spec.persistentVolumeClaimRetentionPolicy` is an optional field that controls if and how PVCs are deleted during the lifecycle of a StatefulSet. There are two policies that are used.
+
+  - `whenDeleted` is applied when the StatefulSet is deleted.
+  - `whenScaled` is applied when the replica count of the StatefulSet is reduced, for example when scaling down the set.
+  
+These may each be set to either `Delete` or `Retain`.
+
+  - `Delete`. The PVCs created from the StatefulSet `volumeClaimTemplate` are deleted, for each Pod affected by the policy. With the `whenDeleted` policy, as all Pods are deleted when the Statefulset is deleted, all PVCs from the `volumeClaimTemplate` are deleted. With the `whenScaled` policy, only PVCs corresponding to Pod replicas being scaled down are deleted. Note PVC deletion only happens for this particular kind of Pod deletion. For example, Pods that are deleted due to being killed for rescheduling on a different node are not affected, and the PVCs will live to migrate to the new node.
+  - `Retain`. PVCs from the `volumeClaimTemplate` are not affected when their Pod is deleted. This is the behavior before this new feature.
+  
+The default for policies is `Retain`, matching the StatefulSet behavior before this new feature.
+
+Here is an example policy.
+
+```
```

Comment:
nit: append `yaml` to mark this as YAML.

Reply:
done